src/Distributions/BernoulliExpNeg/Correctness.dfy

```diff
@@ -163,4 +171,321 @@ module BernoulliExpNeg.Correctness {
     requires 0 <= gamma.numer <= gamma.denom
     ensures Independence.IsIndep(Model.SampleLe1(gamma))
 
+
+  lemma Le1LoopIterCorrectness(gamma: Rationals.Rational, k: nat)
```

Comment:
I'd appreciate a few more explanatory comments in this file.

Reply:
I added comments to the lemmas.